node/src/node/app.cpp

```diff
@@ -657,7 +657,9 @@ app_t::app_t(context_t& context,
     work(std::make_unique<asio::io_service::work>(*loop)),
     thread(nullptr)
 {
-    const profile_t profile{context, profile_name};
+    auto manifest = manifest_t(context, name);
+    auto extended_info = manifest.extended_info;
+    const profile_t profile{context, profile_name, extended_info};
```

Comment:
It's extra copying here.
```
    auto manifest = manifest_t(context, name);
    const auto& extended_info = manifest.extended_info;
    const profile_t profile{context, profile_name, extended_info};
```
or
```
    auto manifest = manifest_t(context, name);
    const profile_t profile{context, profile_name,  manifest.extended_info};
```
will be better.

Reply:
tnx, fixed